### PR TITLE
Made a blog for solution to question of Day 1

### PR DESCRIPTION
**Pull request by marshallKR7**

### CODE REVIEW

**Comment by marshallKR7** [comment]
@gauthamzz there is 50 points for making blog,isn't it?
Can you clarify me please though you have stated it before.

**Review by gauthamzz — CHANGES_REQUESTED** [review]
That’s not how you highlight code in markdown , google and correct it

**Comment by marshallKR7** [comment]
Can you please explain a little more about what i have to change? That would be helpful. :D

File: _posts/2018-01-13-Day-1-Question-and-Solution.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## **Input:**
+## **The first line of the input contains a single integer T, denoting the number of test cases. Then T test case follows, a single line of the input containing three integers X,Y and P.**
+
+## **Constraints:**
+## **1<=T<=100**
+## **1<=X<=Y<=P<=1000**
+
+## **One of the solution:**
+	
+### package main
```
Comment (mubaris) [inline _posts/2018-01-13-Day-1-Question-and-Solution.md]: `###` is for Heading as well any number of hashed from 1 to 6.

For code highlighting you have to use tilda charcter `

Read More about markdown from here - [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

Comment (mubaris) [reply _posts/2018-01-13-Day-1-Question-and-Solution.md]: Highlight code properly and Change `##` proper paragraphs

**Review by mubaris — CHANGES_REQUESTED** [review]

**Comment by marshallKR7** [comment]
I have made the changes.Please see it.

File: _posts/2018-01-13-Day-1-Question-and-Solution.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+published: true
```
Comment (mubaris) [inline _posts/2018-01-13-Day-1-Question-and-Solution.md]:
```
---
published: true
author: <username>
layout: post
title: Even-Odd turn game Solution in Go
---
```
You need to add these many info

**Review by mubaris — CHANGES_REQUESTED** [review]

**Comment by marshallKR7** [comment]
@mubaris updated it. :D

**Comment by gauthamzz** [comment]
well anyway one post is merged. This wont be merged. But you will get points.

**Comment by marshallKR7** [comment]
Yup its ok but i think there was some confusion as you requested me the changes in blog and after 2 hours other contestant's PR on the same issue was merged.

**Pull request closed without merging**